apps/estuary/test/unit/event_table_test.exs

```diff
@@ -1,48 +0,0 @@
-defmodule Estuary.InsertTest do
```

Comment:
Why did you delete this file?

Reply:
These tests are redundant now. Either from other unit tests, the E2E test, or the pipeline tests.